helpers/deploymentConfig.ts

```diff
@@ -240,6 +241,13 @@ export const chainlinkFeed: Config = {
     ARB: "0xD1092a65338d049DB68D7Be6bD89d17a0929945e",
     WETH: "0xd30e2101a97dcbAeBCBC04F14C3f624E67A35165",
   },
+  arbitrumone: {
```

Comment:
Should we also updated `redstoneFeed`, adding the XVS addresses for arbitrum one (and Ethereum)?

Reply:
Done https://github.com/VenusProtocol/oracle/pull/192/commits/8c1b942ada08033c44c22fe20ab6eeb7062cf09f